homeassistant/components/reolink/sensor.py

```diff
@@ -58,7 +58,16 @@ class ReolinkHostSensorEntityDescription(
         state_class=SensorStateClass.MEASUREMENT,
         entity_category=EntityCategory.DIAGNOSTIC,
         value=lambda api, ch: api.ptz_pan_position(ch),
-        supported=lambda api, ch: api.supported(ch, "ptz_position"),
+        supported=lambda api, ch: api.supported(ch, "ptz_pan_position"),
+    ),
+    ReolinkSensorEntityDescription(
```

Comment:
What would be the value of such a sensor?

And why is this a sense? Should this later be part of an actual PTZ feature if we develop one in the future?

What would be the use case for this even at this point?

Reply:
Value is between 0 and around 3000 and gives the tilt postion of the camera (probably the motor steps, guessing they are using a stepper motor in their cams).

Together with the already existing pan sensor these uniquely define the direction the camera is pointing at.

These cameras have auto-tracking capabality so they can start moving "on their own". These sensors will then show you where the cameras is at. Could be used in conditions of automations.

Currently the values of these sensors are beeing polled every 1 minute.
But their is a clear path to get these values as TCP push updates (requires new firmware), in which case it will emidiatly show if the camera is moving.

And you are right: I am discussing with Reolink to get the possibility to set the camera position using these same coordinates. But that is not yet available in the firmware.
Once there is firmare supporting this, I will convert these sensors into number entities with sliders wich will allow you to set the absolute position of the camera.

Currently you can only start continues movement left, right, up, and down after which you have to press "stop".
And there are preset points available. (limited to 64 points and have to be set-up in the reolink app).
But no way to set the absolute postion of the camera directly.